<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LRUCache</class>  <!-- 类名改为LRUCache -->
 <widget class="QMainWindow" name="LRUCache">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Cache Demo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="urlEdit">
        <property name="placeholderText">
         <string>Enter image URL...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadButton">
        <property name="text">
         <string>Load Image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>Clear Cache</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="imageLabel">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string>Ready</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>